config/network-policy/allow-metrics-traffic.yaml

```diff
@@ -0,0 +1,26 @@
+# This NetworkPolicy allows ingress traffic
+# with Pods running on namespaces labeled with 'metrics: enabled'. Only Pods on those
+# namespaces are able to gathering data from the metrics endpoint.
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/name: output-dir
```

Comment:
nit pick: I wonder if we want to add a label as `app.kubernetes.io/name: output-dir`

Reply:
updated!